### PR TITLE
Add voltage init alert threshold on shunt compensator activation

### PR DESCRIPTION
**Pull request by FranckLecuyer**

### CODE REVIEW

File: src/components/dialogs/parameters/widget/parameter-float.tsx [hunk 1]
```diff
@@ -14,24 +14,35 @@ export interface ParameterFloatProps {
...
 }: ParameterFloatProps) => {
     return (
         <>
-            <Grid item container direction={'row'} spacing={1} paddingTop={3}>
-                <Grid item xs={10} sx={style}>
-                    <FormattedMessage id={label} />
-                </Grid>
-                <Grid item xs={2}>
-                    <FloatInput name={name} adornment={adornment} />
+            <Tooltip title={tooltip ? <FormattedMessage id={tooltip} /> : ''}>
```
Comment (jonenst) [inline src/components/dialogs/parameters/widget/parameter-float.tsx:29]: don't put Tooltip component if tooltip is not defined ?

File: src/components/dialogs/parameters/widget/parameter-float.tsx [hunk 2]
```diff
@@ -14,24 +14,35 @@ export interface ParameterFloatProps {
...
 export const ParameterFloat = ({
     name,
     label,
     style,
     adornment,
+    tooltip,
 }: ParameterFloatProps) => {
     return (
         <>
-            <Grid item container direction={'row'} spacing={1} paddingTop={3}>
```
Comment (jonenst) [inline src/components/dialogs/parameters/widget/parameter-float.tsx:27]: previous grid was an item, it means the parent (parent of this component) was a grid container. It should not be ok to replace it with the Tooltip component?

File: src/components/dialogs/parameters/widget/parameter-float.tsx (hunk 1 above)
Comment (FranckLecuyer) [reply src/components/dialogs/parameters/widget/parameter-float.tsx]: Done

File: src/components/dialogs/parameters/widget/parameter-float.tsx (hunk 2 above)
Comment (FranckLecuyer) [reply src/components/dialogs/parameters/widget/parameter-float.tsx:27]: Done

**Review by ayolab — APPROVED** [review]

**Pull request merged**